Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/Lights/Ltc.azsli

```diff
@@ -76,47 +76,43 @@ float3x3 BuildViewAlignedOrthonormalBasis(in float3 normal, in float3 dirToView)
     return float3x3(tangent, bitangent, normal);
 }
 
+// The following two edge integration functions are based on the work from:
+// [HILL16] Real-Time Area Lighting: a Journey from Research to Production, SIGGRAPH 2016
+// Ref: https://blog.selfshadow.com/publications/s2016-advances/
+
 // Cosine integration of edge in a hemisphere. v1 and v2 should be normalized vertices above the
 // xy plane in positive z space.
 float IntegrateEdge(float3 v1, float3 v2)
 {
-    // This alternate version may work better for platforms where acos() precision is low.
-    /*
-    float x = dot(v1, v2);
-    float y = abs(x);
+    float cosTheta = dot(v1, v2);
+    float absCosTheta = abs(cosTheta);
 
-    float a = 5.42031 + (3.12829 + 0.0902326 * y) * y;
-    float b = 3.45068 + (4.18814 + y) * y;
+    // Cubic rational fitting of x/sin(x)
+    float a = 5.42031 + (3.12829 + 0.0902326 * absCosTheta) * absCosTheta;
+    float b = 3.45068 + (4.18814 + absCosTheta) * absCosTheta;
     float theta_sinTheta = a / b;
 
-    if (x < 0.0)
+    if (cosTheta < 0.0)
     {
-        theta_sinTheta = PI * rsqrt(saturate(1.0 - x * x)) - theta_sinTheta;
+        theta_sinTheta = PI * saturate(rsqrt(saturate(1.0 - cosTheta * cosTheta))) - theta_sinTheta;
```

Comment:
Hey I realize my suggestion from the bug wasn't correct here. The rsqrt of a 0-1 number will always be greater than 1, so the outer saturate is always going to clamp it to 1. I'd expect this to really throw off the results. I think we'll just have to eat the minor cost of a max() here.
```theta_sinTheta = PI * max(1.0, rsqrt(saturate(1.0 - cosTheta * cosTheta))) - theta_sinTheta;```

Reply:
**On why I turned on the rational fitting by default.**
The rational fitting approach is the one that was recommended in the paper to address precision issues caused by acos not being a proper transcendental operation but just a proxy for code gen. (I also confirmed that the implementation given in the paper is still used).  
The reason I decided to activate that code path, is to unify the behaviour. That way we have an operation that will execute with similar precision on every targets, minimizing the chance of visual difference between targets.
On the performance side, I could not observe any meaningful difference, but the fitting approach uses one less VGPR.

**On the saturate.**
Good catch! I ran the tests and couldn't spot any difference between with and without. I'll rework that part and update the review.